docs: 翻訳ガイドラインを作成 (#18)
typst-jp/typst-jp.github.io: README.en.md

# Typst Japanese Documentation (Unofficial)

> 日本語版は[README.md](README.md)を参照してください。

This is an unofficial Japanese translation of the documentation for the typesetting system [Typst](https://typst.app/docs). It has been created with the permission of [Typst GmbH](https://typst.app/legal/).

The repository was forked from the [Chinese version](https://github.com/typst-doc-cn/typst-doc-cn.github.io) and is translated into Japanese based on the official documentation of [Typst v0.11.1](https://typst.app/docs/changelog/#v0.11.1) as of June 2024.

The actual working web version can be viewed at the following URL.
> https://typst-jp.github.io/docs/

## How to participate in translation

> Please refer to the [Translation Guidelines (CONTRIBUTING.md)](CONTRIBUTING.md) and create a [Pull Request](https://github.com/typst-jp/typst-jp.github.io/pulls). Questions and discussions are also welcome in [Issues](https://github.com/typst-jp/typst-jp.github.io/issues).

Reviewer: ```suggestion
Please refer to the [Translation Guidelines (CONTRIBUTING.md)](CONTRIBUTING.md) and create a [Pull Request](https://github.com/typst-jp/typst-jp.github.io/pulls). Any questions or discussions are also welcome in [Issues](https://github.com/typst-jp/typst-jp.github.io/issues).
```

> If you have any questions, please join the Discord server [Kumihan Club](https://discord.gg/9xF7k4aAuH) and contact us.

Reviewer: ```suggestion
Also, feel free to join our Discord server [Kumihan Club](https://discord.gg/9xF7k4aAuH) and contact us.
```

Author: ```suggestion
Also, feel free to join our Discord server [Kumihan Club](https://discord.gg/9xF7k4aAuH) to get in touch with us directly.
```
